docs/docs/event-bus.md

```diff
@@ -5,7 +5,8 @@ title: Event Bus
 # Event Bus
 
 Marathon has an internal event bus that captures all API requests and scaling events. This is useful for integrating with load balancers, keeping stats, etc.
```

Comment:
By subscribing to the event bus, you get informed about every change instantly (without pulling).  
All entities that act on the state of Marathon (e.g. load balancer) benefit from this functionality.